### PR TITLE
setUserProfileTwice

### PR DESCRIPTION
**Pull request by OfirZinkerz**

### CODE REVIEW

File: src/components/user/services/userProfile.service.js [hunk 1]
```diff
@@ -70,19 +70,34 @@ angular.module('znk.infra.user').service('UserProfileService',
...
                 });
             } else {
                 $log.error('UserProfileService.setProfile: No user were found');
                 return $q.when(null);
             }
         }
 
+        function _setUserProfileTwice(profilePath, newProfile) {
+            return InfraConfigSrv.getGlobalStorage().then(function(globalStorage) {
+                return globalStorage.set(profilePath, newProfile);
```
Comment (assafshp) [inline src/components/user/services/userProfile.service.js]: are you sure global storage writes to root ? (not znk-dev ?)

File: src/components/user/services/userProfile.service.js [hunk 2]
```diff
@@ -70,19 +70,34 @@ angular.module('znk.infra.user').service('UserProfileService',
...
-                    return profile ? UserStorageService.update(profilePath, newProfile) : UserStorageService.set(profilePath, newProfile);
+                    if (ENV.setUserProfileTwice) {
+                        saveProfileProm.push(_setUserProfileTwice(profilePath, newProfile));
+                    }
+                    if (profile){
+                        saveProfileProm.push(UserStorageService.update(profilePath, newProfile));
+                    } else {
+                        saveProfileProm.push(UserStorageService.set(profilePath, newProfile));
+                    }
+                    return $q.all(saveProfileProm);
```
Comment (assafshp) [inline src/components/user/services/userProfile.service.js]: who uses the return value ? in the previous code, you returned the profile....here you'll return an array

File: src/components/user/services/userProfile.service.js (hunk 1 above)
Comment (OfirZinkerz) [inline src/components/user/services/userProfile.service.js]: I know that's way it's called _setUserProfileTwice.
once to UserProfileService (znk-dev) twice to globalStorage

File: src/components/user/services/userProfile.service.js (hunk 2 above)
Comment (OfirZinkerz) [inline src/components/user/services/userProfile.service.js]: LoginAppSrv

**Pull request merged**